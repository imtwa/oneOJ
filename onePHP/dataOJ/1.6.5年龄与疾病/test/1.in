19
38 93 16 27 43 89 99 65 98 25 39 48 59 75 37 3 36 25 43 