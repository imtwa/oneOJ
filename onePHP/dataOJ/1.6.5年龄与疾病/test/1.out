10.53%
15.79%
42.11%
31.58%